sky/backends/cloud_vm_ray_backend.py

```diff
@@ -2102,7 +2101,7 @@ def _provision(self,
                         raise exceptions.ResourcesUnavailableError(
                             error_message) from None
             if dryrun:
-                return
+                return None
```

Comment:
I added the return None for the `LocalDockerBackend._provision` as well, but for the return statement at L2277, the method has a return type annotation as `-> None`, so it seems to be fine (or more intuitive) to not having the return value in the return statement.

Seems the standard for being consistent for the return statement has a scope of each function, IIUC. It might be fine to only have the return statement consistent in a single function.
> Either all return statements in a function should return an expression, or none of them should.